perceval/backends/core/github.py

```diff
@@ -388,9 +435,9 @@ def __init_extra_issue_fields(self, issue):
 
         issue['user_data'] = {}
         issue['assignee_data'] = {}
-        issue['assignees_data'] = {}
-        issue['comments_data'] = {}
-        issue['reactions_data'] = {}
+        issue['assignees_data'] = []
```

Comment:
Not sure whether to initialize all these keys is better to use `setdefaults` rather than do it manually. What do you think?

Reply:
Probably *setdefaults* is a cleaner solution